BluePic-Server/Sources/Server/RouteUtils.swift

```diff
@@ -277,7 +269,7 @@ func updateImageJSON(json: JSON, withRequest request: RouterRequest) throws -> J
  - returns: NSError object
  */
 func generateInternalError() -> NSError {
-  return NSError(domain: BluePic.Domain, code: BluePic.Error.Internal.rawValue, userInfo: [NSLocalizedDescriptionKey: String(BluePic.Error.Internal)])
+  return NSError(domain: BluePic.Domain, code: BluePic.Error.Internal.rawValue, userInfo: [NSLocalizedDescriptionKey: String(describing: BluePic.Error.Internal)])
```

Comment:
Yeah, `LocalizedError` is a protocol, therefore, it needs an instance- an enum, struct, or class. I recommend overriding the `errorDescription: String?` property for it to work properly. `NSError` is deprecated, but still works for now....